.github/workflows/build.yml

```diff
@@ -136,8 +151,8 @@ jobs:
 
       - name: Prepare .env for Examples
         run: |
-          echo "OPERATOR_KEY=0xa608e2130a0a3cb34f86e757303c862bee353d9ab77ba4387ec084f881d420d4" > examples/.env
-          echo "OPERATOR_ID=0.0.1022" >> examples/.env
+          echo "OPERATOR_KEY=${{ secrets.OPERATOR_KEY }}" > examples/.env
```

Comment:
Seems it is not working, because tests and examples are failing now with `PAYER_ACCOUNT_NOT_FOUND`

Reply:
reverted